### ISSUE TITLE
cert expired for *.poshtiban.io

### ISSUE THREAD

**Issue by j0lol** [Gictorbit/photoshopCClinux#157]
![image](https://user-images.githubusercontent.com/24716467/153269114-1555d6d1-dd0c-4277-9acc-e0193b56c228.png)

**Comment by No-Biggie805** [comment]
Hi during instalation my pc also reports error about certificate, could it be related smh?

**Comment by j0lol** [comment]
> Hi during instalation my pc also reports error about certificate, could it be related smh?

yeah replace line 197 in `scripts/sharedFuncs.sh` to `curl --insecure $3 -o $1` to fix that (and dont use aria2)

**Comment by No-Biggie805** [comment]
Yeah it worked thx!!
